packages/calypso-build/CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 ## trunk
 
+- Breaking: Changed `webpack/file-loader` to internally use the new `asset/resource` modules. The semantics of
```

Comment:
Is it worth mentioning the reason (deprecation since Webpack 5) for this change?